fe/fe-core/src/main/java/com/starrocks/catalog/PhysicalPartitionImpl.java

```diff
@@ -53,8 +56,9 @@ public class PhysicalPartitionImpl extends MetaObject implements PhysicalPartiti
     @SerializedName(value = "parentId")
     private long parentId;
 
+    // DEPRECATED
     @SerializedName(value = "shardGroupId")
-    private long shardGroupId;
+    private long shardGroupId = INVALID_SHARD_GROUP_ID;
```

Comment:
Default values ​​for serialized properties need to add an empty constructor, otherwise the default value will be null after upgrading from an older version.